gensim/summarization/bm25.py

```diff
@@ -18,7 +18,8 @@ class BM25(object):
 
     def __init__(self, corpus):
         self.corpus_size = len(corpus)
-        self.avgdl = sum(map(lambda x: float(len(x)), corpus)) / self.corpus_size
+        self.avgdl = sum(map(lambda x: float(len(x)), corpus)
```

Comment:
We ignore the 79 chars guideline. Put line breaks in places that are conceptually and visually meaningful. Try not to exceed 120 characters per line (unless necessary).